src/refractor/prism.css

```diff
@@ -60,6 +60,17 @@ code[class*="language-"]::selection, code[class*="language-"] ::selection {
 	background: #073642; /* base02 */
 }
 
+pre[class="language-text"],
+code[class="language-text"] {
+	color: #073642; /* base02 */
+}
+
+pre[class="language-text"]::selection, pre[class="language-text"] ::selection,
+code[class="language-text"]::selection, code[class="language-text"] ::selection {
+	color: #eee8d5; /* base2 */
+	background: #073642; /* base02 */
+}
```

Comment:
Could you try to create a new file called `src/refractor/prism.overrides.scss` and move the content above in this new file? I'd like not to change the original file if possible (which has been directly taken from the library IIRC)

Reply:
Thanks @willdurand 
I updated it.